tests/test_opds.py

```diff
@@ -88,6 +89,27 @@ def library_catalog(*args, **kwargs):
         small_catalog = small_feed.catalog['catalogs']
         eq_([False], small_catalog)
 
+        # Try it with a query that returns no results. No catalogs
+        # are included at all.
+        small_feed = Mock(self._db, "title", "url", self._db.query(Library))
+        small_catalog = small_feed.catalog['catalogs']
+        eq_([], small_catalog)
+
+    def _test_feed_is_large(self):
+        # Verify that the _feed_is_large helper method
+        # works whether it's given a Python list or a SQLAlchemy query.
+        setting = ConfigurationSetting.sitewide(
+            self._db, Configuration.LARGE_FEED_SIZE
+        )
+        setting.value = 2
+        m = OPDSCatalog.feed_is_large
+        list = [1,2,3]
```

Comment:
Shouldn't it be `_feed_is_large` with the leading underscore? Also, `list` isn't being used so how do the two tests below give different results?

Reply:
The problems you found are valid; I never finished writing this test. The tests were passing because this test wasn't being run; _its_ name starts with an underscore so nose didn't recognize it as a test.